Sources/TensorFlow/Layers/Convolutional.swift

```diff
@@ -631,3 +631,107 @@ public struct ZeroPadding3D<Scalar: TensorFlowFloatingPoint>: ParameterlessLayer
         return input.padded(forSizes: [padding.0, padding.1, padding.2])
     }
 }
+
+/// A 2-D Separable convolution layer.
+///
+/// This layer performs a depthwise convolution that acts separately on channels followed by
+/// a pointwise convolution that mixes channels.
+@frozen
+public struct SeparableConv2D<Scalar: TensorFlowFloatingPoint>: Layer {
+    /// The 4-D depthwise convolution kernel.
+    public var depthwiseFilter: Tensor<Scalar>
+    /// The 4-D pointwise convolution kernel.
+    public var pointwiseFilter: Tensor<Scalar>
+    /// The bias vector.
+    public var bias: Tensor<Scalar>
+    /// The element-wise activation function.
+    @noDerivative public let activation: Activation
+    /// The strides of the sliding window for spatial dimensions.
+    @noDerivative public let strides: (Int, Int)
+    /// The padding algorithm for convolution.
+    @noDerivative public let padding: Padding
+
+    /// The element-wise activation function type.
+    public typealias Activation = @differentiable (Tensor<Scalar>) -> Tensor<Scalar>
+
+    /// Creates a `SeparableConv2D` layer with the specified depthwise and pointwise filter,
+    /// bias, activation function, strides, and padding.
+    ///
+    /// - Parameters:
+    ///   - depthwiseFilter: The 4-D depthwise convolution kernel.
+    ///   - pointwiseFilter: The 4-D pointwise convolution kernel.
+    ///   - bias: The bias vector.
+    ///   - activation: The element-wise activation function.
+    ///   - strides: The strides of the sliding window for spatial dimensions.
+    ///   - padding: The padding algorithm for convolution.
+    public init(
+        depthwiseFilter: Tensor<Scalar>,
+        pointwiseFilter: Tensor<Scalar>,
+        bias: Tensor<Scalar>,
+        activation: @escaping Activation = identity,
+        strides: (Int, Int) = (1, 1),
+        padding: Padding = .valid
+    ) {
+        self.depthwiseFilter = depthwiseFilter
+        self.pointwiseFilter = pointwiseFilter
+        self.bias = bias
+        self.activation = activation
+        self.strides = strides
+        self.padding = padding
+    }
+
+    /// Returns the output obtained from applying the layer to the given input.
+    ///
+    /// - Parameter input: The input to the layer.
+    /// - Returns: The output.
+    @differentiable
+    public func callAsFunction(_ input: Tensor<Scalar>) -> Tensor<Scalar> {
+        depthwise = depthwiseConv2D( input,
```

Comment:
```suggestion
        depthwise = depthwiseConv2D(
            input,
```
and then also indent the other arguments properly.